sable_ircd/info.txt

```diff
@@ -0,0 +1,21 @@
+Sable --
+
+Copyright (c) 2021-2024 Sable Development Team
```

Comment:
is this the correct copyright assignment?

Reply:
Good question. It's generic enough to serve the purpose for now.